### ISSUE TITLE
font from schema not respected in code?

### ISSUE THREAD

**Issue by fossfreedom** [linuxmint/slick-greeter#50]
Quick glance through the code:

 https://github.com/linuxmint/slick-greeter/search?utf8=%E2%9C%93&q=font+ubuntu&type=

Whilst the schema has a "font to use" type key - from the above search there are several instances where "ubuntu" is hard-coded.

**Comment by gabrc52** [comment]
Yes, the font config only seems to apply to the top bar. Everything else uses Ubuntu or fallback DejaVu Sans font.

**Comment by nstommel** [comment]
@fossfreedom Yeah, this is a mess. Hardcoded fonts are a big disappointment.

**Comment by icarter09** [comment]
does anyone have any updates on this ticket?

**Comment by gabrc52** [comment]
Still not fixed

**Comment by icarter09** [comment]
@gabrc52 thanks for the update

**Comment by fossfreedom** [comment]
@icarter09 the version and distro is immaterial to your query. As I stated when I raised this, the font usage is hard coded in the source. For non ubuntu distros that don't have the ubuntu fonts the greeter doesn't look as good as it should due to the hard coded font name usage.

**Comment by holisticode** [comment]
I can confirm this. Using manjaro with lightdm and slick-greeter version `local/lightdm-slick-greeter 1.3.2-1`  whatever I change `font-name` to is ignored - for me even on the top bar. Nothing changes.

**Comment by ClaytonTDM** [comment]
still an issue, insane
system info: https://termbin.com/9ix6
